README.md

```diff
@@ -60,7 +60,7 @@ $ npm install -g @salesforce/commerce
 $ sfdx COMMAND
 running command...
 $ sfdx (-v|--version|version)
-@salesforce/commerce/248.1.3 darwin-x64 node-v18.16.1
+@salesforce/commerce/250.0.0 darwin-x64 node-v18.16.1
```

Comment:
Is there a way to Use the release value from package.json

Reply:
@kalagarraj  We already automated this. README is updated automatically with version changes.